script/rename.js

```diff
@@ -115,7 +115,7 @@
     // attach handler to mobile menu entry
     const $mobileMenuOption = jQuery('form select[name=do] option[value=plugin_move]');
     if ($mobileMenuOption.length === 1) {
-        bindFirst($mobileMenuOption.form().find('select[name=do]'), 'change', function (e) {
+        bindFirst($mobileMenuOption.closest('form').find('select[name=do]'), 'change', function (e) {
```

Comment:
```suggestion
        bindFirst($mobileMenuOption.closest('select[name=do]'), 'change', function (e) {
```

According to the selector above, the select is between the option and the form in the hierarchy (which makes sense), so this can be simplified.

Reply:
Ah! You're right of course. I amended my commit accordingly.